From 76284b339884d55c66610de82bb446109351e280 Mon Sep 17 00:00:00 2001
From: RubenKelevra <ruben@vfn-nrw.de>
Date: Sat, 14 May 2016 17:55:18 +0200
Subject: [PATCH 1/1] backport: ipq806x/base-files - Add Archer C2600 board
 LEDs/network

Signed-off-by: Josh Bendavid <joshbendavid@gmail.com>
[darryl@sokoloski.ca: submitting patches on behalf of Josh Bendavid]
Signed-off-by: Darryl Sokoloski <darryl@sokoloski.ca>

source: https://patchwork.ozlabs.org/patch/612488/
---
 target/linux/ipq806x/base-files/etc/board.d/01_leds    | 7 +++++++
 target/linux/ipq806x/base-files/etc/board.d/02_network | 1 +
 2 files changed, 8 insertions(+)

diff --git a/target/linux/ipq806x/base-files/etc/board.d/01_leds b/target/linux/ipq806x/base-files/etc/board.d/01_leds
index b32c1cc..f5b4151 100755
--- a/target/linux/ipq806x/base-files/etc/board.d/01_leds
+++ b/target/linux/ipq806x/base-files/etc/board.d/01_leds
@@ -11,6 +11,13 @@ board_config_update
 board=$(ipq806x_board_name)
 
 case "$board" in
+c2600)
+        ucidef_set_led_usbdev "usb1" "USB 1" "usb_2:blue" "2-1"
+        ucidef_set_led_usbdev "usb2" "USB 2" "usb_4:blue" "4-1"
+        ucidef_set_led_netdev "wan" "WAN" "wan:blue" "eth0"
+        ucidef_set_led_netdev "lan" "LAN" "lan:blue" "br-lan"
+        ucidef_set_led_default "general" "general" "ledgnr:blue" "1"
+        ;;
 d7800 |\
 r7500)
 	ucidef_set_led_usbdev "usb1" "USB 1" "r7500:white:usb1" "1-1"
diff --git a/target/linux/ipq806x/base-files/etc/board.d/02_network b/target/linux/ipq806x/base-files/etc/board.d/02_network
index 6e18297..e3c9137 100755
--- a/target/linux/ipq806x/base-files/etc/board.d/02_network
+++ b/target/linux/ipq806x/base-files/etc/board.d/02_network
@@ -13,6 +13,7 @@ board=$(ipq806x_board_name)
 
 case "$board" in
 ap148 |\
+c2600 |\
 d7800 |\
 r7500)
 	ucidef_add_switch "switch0" \
-- 
2.8.2

